offlineimap/folder/IMAP.py

```diff
@@ -78,9 +87,17 @@ def __selectro(self, imapobj, force=False):
         :param: Enforce new SELECT even if we are on that folder already.
         :returns: raises :exc:`OfflineImapError` severity FOLDER on error"""
         try:
-            imapobj.select(self.getfullname(), force = force)
+            imapobj.select(self.getfullIMAPname(), force=force)
         except imapobj.readonly:
-            imapobj.select(self.getfullname(), readonly = True, force = force)
+            imapobj.select(self.getfullIMAPname(), readonly=True, force=force)
+
+
+    def getfullIMAPname(self):
```

Comment:
Too many blank lines.

Reply:
Well, see https://github.com/uliska/offlineimap/blame/b70f053a2fb8813fdd79475e11dd7d5bb9bf33f4/offlineimap/folder/IMAP.py#L78. And there's a number of further instances of this. (But of course I'll fix the instances in this file)

Reply:
You're right there is a lot of unexpected in the sources.